### PR TITLE
fix variables in openstack.go to keep camel casing and remove unused var

### PR DESCRIPTION
**Pull request by NickrenREN**

### CODE REVIEW

File: pkg/cloudprovider/providers/openstack/openstack.go [hunk 1]
```diff
@@ -46,7 +46,6 @@ const ProviderName = "openstack"
 var ErrNotFound = errors.New("Failed to find object")
 var ErrMultipleResults = errors.New("Multiple results where only one expected")
 var ErrNoAddressFound = errors.New("No address found for host")
-var ErrAttrNotFound = errors.New("Expected attribute not found")
```
Comment (zhouhaibing089) [inline pkg/cloudprovider/providers/openstack/openstack.go]: well, why remove this?

Comment (NickrenREN) [inline pkg/cloudprovider/providers/openstack/openstack.go]: it is never used

**Comment by NickrenREN** [comment]
@k8s-bot ok to test

**Comment by xsgordon** [comment]
@idvoretskyi can you add the sig/openstack label?

**Review by anguslees — APPROVED** [review]
/approve

**Comment by NickrenREN** [comment]
@anguslees  Thanks.Could you help add lgtm and release-note-none labels?

**Comment by anguslees** [comment]
/lgtm

I'm only allowed to cause code to merge, something like release-note-none labels is way above my pay grade :/

**Comment by anguslees** [comment]
@mikedanese for release-note-none rubber-stamping (Mike, please point me somewhere else if there's a more appropriate label manipulation "robot" I should be using).

**Comment by mikedanese** [comment]
@anguslees try that

**Comment by NickrenREN** [comment]
@k8s-ci-robot Could you please add approve label according to the comments here?(Since @anguslees has given /approve )

**Pull request merged**